# Sequence similarity searching

## BLAST

![Extreme-value distribution](https://github.com/Claualvarez/Introdutcion_to_computational_molecular_evolution/blob/master/slides/nihms519883f1.jpg)
**Extreme-value distribution.** The human dual specificity protein phosphatase 12 (DUS12_HUMAN) was compared to 38,114 human RefSeq proteins using the SSEARCH program. The distribution of bit-scores for all 38,114 alignments is shown (squares, â–¡) as well as the mathematically expected distribution of normalized similarity scores (z-scores, or standard deviations above and below the mean 0) based on the size of the database, using the extreme-value distribution. The close agreement between the observed and expected distribution of scores reflects the observation that the distribution of unrelated sequence scores is indistinguishable from random (mathematically generated) scores, so sequences with significant sequence similarity can be inferred to be not-unrelated, or homologous.
> Pearson. Curr Protoc Bioinformatics. 2013

**BLOSUM62 Substitution matrix**

![BLOSUM62 Substitution matrix](https://github.com/Claualvarez/Introdutcion_to_computational_molecular_evolution/blob/master/slides/BLOSUM62.png)

**Activity: sequence similarity searching**
   
   [NCBI](https://blast.ncbi.nlm.nih.gov/Blast.cgi)
   
   [UniProt](https://www.uniprot.org/blast/)
   
   [KEGG](https://www.genome.jp/tools/blast/)
   
   [PDB](https://www.rcsb.org/pdb/search/advSearch.do?st=SequenceQuery)
   
**Position specific scoring matrices**

